### PR TITLE
PubMatic bid adapter: support for FPD 

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
## Type of change
- [ X ] Feature

## Description of change
Adding support in PubMatic bid adapter for First-Party Data

### CODE REVIEW

File: modules/pubmaticBidAdapter.js [hunk 1]
```diff
@@ -998,6 +1030,15 @@ export const spec = {
...
 
+    // First Party Data
+    const commonFpd = config.getConfig('ortb2') || {};
+    if (commonFpd.site) {
+      utils.mergeDeep(payload, {site: commonFpd.site});
+    }
+    if (commonFpd.user) {
+      utils.mergeDeep(payload, {user: commonFpd.user});
+    }
+
```
Comment (patmmccann) [inline modules/pubmaticBidAdapter.js:1041]: why not device?

Comment (patmmccann) [reply modules/pubmaticBidAdapter.js:1041]: fpd module may have several writes to device object #6099

Comment (patmmccann) [reply modules/pubmaticBidAdapter.js:1041]: update: harshad slacked me to indicate a future pr might include device if it becomes popular to set and that the PBS adapter also lacks device.

**Comment by patmmccann** [comment]
LGTM 

Thanks for your comments @pm-harshad-mane ; we've looked this over to confirm it would pass you adslot id with the preauction module installed

**Review by patmmccann — APPROVED** [review]

**Review by jsnellbaker — APPROVED** [review]

**Comment by pm-harshad-mane** [comment]
Thank you @patmmccann , @jsnellbaker

**Pull request merged**